### DISCUSSION TITLE
Playwright Implementation

### DISCUSSION THREAD

**Discussion by Vinyzu** [QIN2DIM/hcaptcha-challenger#214] [Show and tell]
I implemented your AI in my botting studio, Botright.
For those who wanna use this project in playwright its defenetely interesting.

You can find the project here: https://github.com/Vinyzu/Botright
(I also made it undetected and smooth mouse and everything ;:D)

**Comment by QIN2DIM** [comment]
It's a very interesting project. But I would like to know if you are familiar with Playwright's anti-detection approach. I used the solution in undetected_chromedriver directly in this project. But from my experience, Playwright-stealth doesn't provide good camouflage if you don't have your own implementation.

**Reply by Vinyzu** [reply]
Mh, seems like you know what ur talking about. Do you have Discord or any other direct messanger?

**Comment by QIN2DIM** [comment]
I had planned to rewrite this project in Playwright a long time ago, but the underlying principles of Selenium and Playwright are very different, and it seemed very difficult to make a uniform disguise on Playwright.
